modules/tile-converter/src/converter-cli.ts

```diff
@@ -350,7 +360,7 @@ function getIntegerValue(index: number, args: string[]): number {
 
 function getBooleanValue(index: number, args: string[]): boolean {
   const stringValue: string = getStringValue(index, args).toLowerCase().trim();
-  if (args[index] === '--no-draco' && !stringValue) {
+  if (['--no-draco', '--split-nodes'].includes(args[index]) && !stringValue) {
```

Comment:
This seems like a non-obvious use of `includes`. Does it even work as intended? I would recommend a unit test for this if you keep as is.

 I would split into two tests (not sure if && is the logic you want)
```suggestion
  if (args.includes('--no-draco') && args.includes('--split-nodes') && !stringValue) {
```

Reply:
sequence (index) is important